.github/workflows/android-maestro.yml

```
@@ -70,23 +118,25 @@ jobs:
+        runs-on: [self-hosted, macOS, ARM64, macos-maestro]
+        timeout-minutes: 60
         strategy:
             fail-fast: false
             matrix:
                 target: [bare, expo]
         env:
             APP_DIR: packages/react-native-payments-example/apps/${{ matrix.target }}
-            MAESTRO_DEBUG_OUTPUT_DIRECTORY: ${{ github.workspace }}/packages/react-native-payments-example/artifacts/maestro-android-${{ matrix.target }}
 
         steps:
             -   name: Checkout repository
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)